.github/workflows/quality.yaml

```diff
@@ -0,0 +1,95 @@
+name: Quality
+
+on:
```

Comment:
I guess we also want to run it on pushes to master, right?

Reply:
Ok, I will enable it for pushs on master too